lunaria.config.json

```
@@ -0,0 +1,57 @@
+{
+	"$schema": "./node_modules/@lunariajs/core/config.schema.json",
+	"repository": {
+		"name": "solidjs/solid-docs-next",
+		"branch": "main",
+		"hosting": "github"
+	},
+	"defaultLocale": {
+		"label": "English",
+		"lang": "en"
+	},
+	"files": [
+		{
+			"type": "universal",
+			"location": "src/routes/**/*.{mdx,md,tsx}",
+			"pattern": "src/routes/@lang/@path"
+		},
+		{
+			"location": "src/i18n/dictionaries/**/ui.ts",
+			"pattern": "src/i18n/dictionaries/@lang/@path",
+			"type": "dictionary"
+		}
+	],
+	"locales": [
+		{
+			"label": "Português do Brasil",
+			"lang": "pt-br"
+		}
+	],
+	"outDir": "./public/i18n-status",
+	"dashboard": {
+		"site": "https://docs.solidjs.com/i18n-status",
+		"title": "Solid Docs Localization Status",
```

I can see you changed the dashboard UI to use `translation` instead of `localization` in other headings, but here `localization` is still used for the dashboard title, since in this case it's really just translation we can keep it consistent.
```suggestion
		"title": "Solid Docs Translation Status",
```